internal/helm/helm.go

```diff
@@ -218,7 +251,13 @@ func GenManifests(
 	} else {
 		helmChart, err = newChartFromCiliumVersion(ciliumVer)
 		if err != nil {
-			return nil, err
+			if !errors.Is(err, fs.ErrNotExist) {
+				return nil, err
+			}
+			helmChart, err = newChartFromRemote(ciliumVer)
+			if err != nil {
+				return nil, err
+			}
 		}
 	}
```

Comment:
nit: I feel like this logic could be extracted in its own `func` to avoid further nesting.